appcenter-crashes/android/src/main/java/com/getcapacitor/plugin/appcenter/crashes/CrashesBase.java

```diff
@@ -1,8 +1,11 @@
 package com.getcapacitor.plugin.appcenter.crashes;
 
+import com.getcapacitor.JSObject;
 import com.microsoft.appcenter.AppCenter;
 import com.microsoft.appcenter.crashes.Crashes;
 
+import java.util.Map;
```

Comment:
Is this needed?

Reply:
Good catch 😄 No it's not needed, I've removed it now.